### ISSUE TITLE
Automatically update template operator based on latest charmcraft init

### ISSUE THREAD

**Issue by jdkandersson** [canonical/charmcraft#828]
There is a GitHub repo (https://github.com/canonical/template-operator) used as a template for many teams when starting new charms. For any changes to the default/ recommended starting point of charms, both the `charmcraft init` command and the template-operator repo have to be updated. This could potentially be improved by unifying the two, for example by include a GitHub action on this repository to automatically send a commit to the template-operator based on the latest output of `charmcraft init`.

**Comment by facundobatista** [comment]
Hello! Thanks for the report. I'm rejecting this issue, because we will not be working on something like this in the foreseeable future, but let me explain why in more detail...

On one hand, the template-operator diverges from `charmcraft init`. It may be providing more boilerplate around current `init` structure, but probably in the future it will change things on how `init` does. On the other hand `charmcraft init` will grow the concept of "profiles", so we will have several templates that the user could choose when running the command (this very feature may lead people to stop using the github's template. Finally, it feels weird to start adding actions in this public library to hit other projects, this may start a journey that I don't want to go through.

So, in short, let's see how the world is structured when the usage of profiles in `charmcraft init` is wide. We may revisit this idea, but as I said above, closing it for now.

Thanks!
